src/providers/seventv/SeventvEmotes.hpp

```diff
@@ -10,9 +10,13 @@
 namespace chatterino {
 
 class Channel;
-struct SeventvEventAPIEmoteAddDispatch;
-struct SeventvEventAPIEmoteUpdateDispatch;
-struct SeventvEventAPIEmoteRemoveDispatch;
+
+namespace seventv::eventapi {
+    struct EmoteAddDispatch;
+    struct EmoteUpdateDispatch;
+    struct EmoteRemoveDispatch;
+}  // namespace seventv::eventapi
+using namespace seventv::eventapi;
```

Comment:
no using namespace in a header file, it leaks to every source file that includes this